app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -125,13 +129,20 @@ def stringify_attr(attribute)
   end
 
   def ial2_data
-    @ial2_data ||= begin
-      if (ial2_session? || ialmax_session?) && active_profile.present?
-        out_of_band_session_accessor.load_pii(active_profile.id)
-      else
-        Pii::Attributes.new_from_hash({})
-      end
-    end
+    @ial2_data ||= out_of_band_session_accessor.load_pii(active_profile.id)
+  end
+
+  def identity_proofing_requested_for_verified_user?
+    return false unless active_profile.present?
+    resolved_authn_context_result.identity_proofing? || resolved_authn_context_result.ialmax?
+  end
+
+  def resolved_authn_context_result
+    @resolved_authn_context_result ||= AuthnContextResolver.new(
+      service_provider: identity&.service_provider,
+      vtr: (JSON.parse(identity.vtr) if identity.vtr.present?),
```

Comment:
this would save a few parens but is otherwise identical
```suggestion
      vtr: identity.vtr.presence && JSON.parse(identity.vtr),
```